src/chef-mover/src/mover_server_admins_global_group_callback.erl

```diff
@@ -0,0 +1,194 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Tyler Cloke <tyler@chef.io>
+%% @copyright 2015 Chef Software, Inc.
+%%
+%% This migration sets up a global group called server-admins.
+%% It also grants READ and CREATE permissions on the users container
+%% to the server-admins group.
+%%
+-module(mover_server_admins_global_group_callback).
+
+-export([
+         migration_init/0,
+         migration_complete/0,
+         migration_type/0,
+         supervisor/0,
+         migration_start_worker_args/2,
+         migration_action/2,
+         next_object/0,
+         error_halts_migration/0,
+         reconfigure_object/2,
+         needs_account_dets/0
+        ]).
+
+-include("mover.hrl").
+-include("mv_oc_chef_authz.hrl").
+
+-define(GLOBAL_PLACEHOLDER_ORG_ID, <<"00000000000000000000000000000000">>).
+
+-record(container, {authz_id}).
+-record(user, {authz_id}).
+-record(mover_chef_group, {
+          id,
+          org_id,	  
+          authz_id,
+          name,
+          last_updated_by,
+          created_at,
+          updated_at
+         }).
+
+
+migration_init() ->
+    mv_oc_chef_authz_http:create_pool(),
+    mover_transient_migration_queue:initialize_queue(?MODULE, [?GLOBAL_PLACEHOLDER_ORG_ID]).
+
+migration_action(GlobalOrgId, _AcctInfo) ->
+    BifrostSuperuserAuthzId = mv_oc_chef_authz:superuser_id(),
+    ErchefSuperuserAuthzId = get_erchef_superuser_authz_id(),
+    ServerAdminsAuthzId = case create_server_admins_authz_group(BifrostSuperuserAuthzId) of
+        AuthzId when is_binary(AuthzId) ->
+            AuthzId;
+        AuthzError ->
+            lager:error("Could not create new authz group for server-admins."),
+            throw(AuthzError)
+    end,
+    
+    case create_server_admins_global_group(ServerAdminsAuthzId, GlobalOrgId) of
+        {chef_sql, {GroupError, _}} ->
+	    lager:error("Could not create new erchef group for server-admins."),
+            throw(GroupError);
+	_ -> true
+    end,
+
```

Comment:
The default permissions are empty on a group created by the superuser. We should add the group to its own read ACE so that members can see who is in it.